WEB-452 Administratory Notifications

fladens opened the pull request:
Feature that provides the ability to broadcast notifications to all users within the admin console.
Admin/FeatureContext does not work with Mink should be possible
Admin/FeatureContext refactoring.

Added a Tool in the admin console to broadcast Notifications.
Added a BroadcastNotification to show the message to the user.
Added a structure to the Context Class with regions for easy collapsation of code and a better overview
Added/Updated PHPDocs to all functions
Fixed the Admin/FeatureContext to work with Mink and Screens

new Doctrine entity:
BroadcastNotification

HCrane reviewed (changes requested):
1. MIgration missing
2. Some small checkstyle errors (please run phpcheckstyle(see confluence testing page for snippet) before pushing to repo)

fladens committed:
WEB-452 Administratory Notifications

Feature that provides the ability to broadcast notifications to all users within the admin console.
Admin/FeatureContext does not work with Mink should be possible
Admin/FeatureContext refactoring and code style fixing.

Added a Tool in the admin console to broadcast Notifications.
Added a BroadcastNotification to show the message to the user.
Added a structure to the Context Class with regions for easy collapsation of code and a better overview
Added/Updated PHPDocs to all functions
Fixed the Admin/FeatureContext to work with Mink and Screens

new Doctrine entity:
BroadcastNotification

fladens commented:
Added Migration and fixed the checkstyle errors. @HCrane
